timeline-chart/src/components/time-graph-row-element.ts

```diff
@@ -55,16 +55,16 @@ export class TimeGraphRowElement extends TimeGraphComponent {
 
     renderLabel() {
         const position = this._options.position;
-        const width = this._options.width;
+        const width = Math.round(this._options.width);
```

Comment:
Any reason why we have to round the width?

Reply:
No specific reason just to make the calculation easier (we compare the text width to the 1/10 th of the state's width) but reading through the code again I realized that by rounding off the width we will wrongfully offset the textObjx position since its a const.